wfl/calculators/orca/basinhopping.py

```diff
@@ -160,10 +160,10 @@ def __init__(self, atoms=None, n_hop=10, n_run=3, scratch_path=None,
         self.chained_hops = chained_hops
         self.uhf = uhf
 
-        if scratch_path is not None:
-            self.scratch_path = scratch_path
+        if scratchdir is not None:
```

Comment:
Do we think that this default should be implemented by `WFLFileIOCalculator`, if possible?

Reply:
This modification came from "replace all". Basin hopping calculator is in need of a larger refactor, but I'd like to discuss with @stenczelt before starting on it.

Reply:
OK, let's leave it alone for now.